lib/canadianVehicle.ts

```diff
@@ -60,21 +60,44 @@ export default class CanadianVehicle extends BaseVehicle {
   }
 
   async unlock(): Promise<any> {
-    logger.info('Begin lock request');
+    logger.info('Begin unlock request');
     const preAuth = await this.getPreAuth();
     const response = await this._request(this.endpoints.unlock, {
       pAuth: preAuth
     });
     return response.body;
   }
 
-  async start(config: StartConfig): Promise<any> {
-    logger.info('Begin start request');
+  async start(
+    airCtrl: Boolean,  // climatisation
```

Comment:
Can we make a config for this instead of passing in multiple params?

Reply:
Uhm , I usually prefer this type of function, but yes, I can do similar to your code, will do

Reply:
```
export interface StartConfig {
  airCtrl?: boolean|string;
  igniOnDuration: number;
  airTempvalue?: number;
  defrost?: boolean|string;
  heating1?: boolean|string;
}
```
airTempvalue is number ? are you sure for US side ?
canadian one is a weird one. 
it is an hex value of the increment of half degrees since 6 degrees
      var airTempValue: String = (6 + (airTemp - 17) * 2).toString(16).toUpperCase() + 'H';

these were my test
        // temp 17-27   06 + (x-17)*2
        // 17   06H     22.5    11H
        // 17.5 07H     23      12H
        // 18   08H     23.5    13H
        // 18.5 09H     24      14H
        // 19   0AH     24.5    15H
        // 19,5 0BH     25      16H
        // 20   0CH     25.5    17H
        // 20.5 0DH     26      18H
        // 21   0EH     26.5    19H
        // 21.5 0FH     27      1AH
        // 22   10H

that's why I mapped into parameters because I cannot feed your struct to body as for US side.
will wait for your input

Reply:
`  body.hvacInfo[airTemp] = {value: airTempValue,unit:0,hvacTempType:1}`

-> airTemp canadian side is a json not a simple number

should be : 
```

export interface AirTemp {
  value: string;
  unit: number;
  hvacTempType: number;
}

export interface StartConfig {
  airCtrl?: boolean|string;
  airTemp?: AirTemp;
  defrost?: boolean|string;
  heating1?: boolean|string;
}
```

Reply:
Oh wow! In that case maybe the solution is create a new interface, say `CandianStartConfig`? Otherwise I'm fine to just allow multiple parameters.

Reply:
I reused your StartConfig and handle the difference in the function.
please review